README

```diff
@@ -1,15 +1,20 @@
-dwm - dynamic window manager
-============================
-dwm is an extremely fast, small, and dynamic window manager for X.
-
-
-Requirements
-------------
+# dde - dynamic desktop environment
```

Comment:
I think we should add a file extension on this, it is clearly using some sort of formatting (looks like markdown).

if markdown, maybe `.md` would be appropriate. This would allow us to later provide linting in CI, and also helps for editor integration (tools such as `.editorconfig` and `markdownlint`)